sys/include/xtimer.h

```diff
@@ -196,233 +201,240 @@ static inline void xtimer_spin(xtimer_ticks32_t ticks);
  * (@p last_wakeup + @p period).
  *
  * This function can be used to create periodic wakeups.
- * @c last_wakeup should be set to xtimer_now() before first call of the
+ * @c last_wakeup_ticks should be set to xtimer_now() before first call of the
  * function.
  *
- * If the result of (@p last_wakeup + @p period) would be in the past, the function
- * sets @p last_wakeup to @p last_wakeup + @p period and returns immediately.
+ * If the result of (@p last_wakeup + @p period) would be in the past, the
+ * function sets @p last_wakeup to @p last_wakeup + @p period and returns
+ * immediately.
  *
- * @param[in] last_wakeup   base time stamp for the wakeup
- * @param[in] period        time in microseconds that will be added to last_wakeup
+ * @param[in] last_wakeup_ticks   base time stamp for the wakeup
+ * @param[in] period_us           time in microseconds that will be added to
+ *                                last_wakeup
  */
-static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup, uint32_t period);
+static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup_ticks,
+                                          uint32_t period_us);
 
 /**
  * @brief Set a timer that sends a message
  *
- * This function sets a timer that will send a message @p offset ticks
- * from now.
+ * This function sets a timer that will send a message @p offset_usec
```

Comment:
Don't know why Doxygen on Travis isn't complaining about it, but here and in other places you are referencing a parameter that does not exist anymore.